### ISSUE TITLE
Write an article about JSS internal processes

### ISSUE THREAD

**Issue by kof** [cssinjs/jss#468]
- How we do we generate CSS rules
- How do we render the rules to the CSSOM
- How do we update
- How do we remove

**Comment by iamstarkov** [comment]
one wish though: can you construct this article as a walk through from writing an sheet object up to it being rendered into styletag and being rerendered

**Comment by kof** [comment]
@iamstarkov yeah, probably this way with examples it will be easier to follow.

**Comment by kof** [comment]
Function values article - done https://medium.com/@oleg008/high-performance-dynamic-styles-db28c873940
